src/sqlfluff/rules/L033.py

```diff
@@ -48,7 +48,13 @@ def _eval(self, context: RuleContext) -> LintResult:
         Note only some dialects have concept of UNION DISTINCT, so rule is only
         applied to dialects that are known to support this syntax.
         """
-        if context.dialect.name not in ["ansi", "bigquery", "hive", "mysql"]:
+        if context.dialect.name not in [
+            "ansi",
+            "bigquery",
+            "hive",
+            "mysql",
+            "redshift",
+        ]:
```

Comment:
Looks good! This makes me wonder if we should have a way to check the parent hierarchy for dialects, so we wouldn't need to explicitly check for `redshift` (it inherits from `postgres`, I think?).

Reply:
Yes but weirdly Postgres DOESN’T support this! So not always that easy.

Reply:
Heh, the "S" in SQL definitely doesn't stand for "standard"!!!